internal/database/prune.go

```diff
@@ -1,24 +1,20 @@
 package database
 
 import (
-	"log/slog"
 	"time"
 )
 
-func DownsampleOverTime(interval time.Duration, database Database) error {
+func DownsampleOverTime(interval time.Duration, downsample_thresh time.Duration, delete_thresh time.Duration, database Database) error {
```

Comment:
It's probably nicer to make a new struct here, so it's clearer at the callsight the args are right.

```go
type PruneConfig struct {
   interval time.Duration
   deleteThreshold time.Duration
   downsampeThreshold time.Duration
}
```